content/ai/concepts/computer-vision/computer_vision.md

```diff
@@ -0,0 +1,86 @@
+---
+Title: 'AI for Computer Vision'
+Description: 'Computer vision is a subfield of artificial intelligence, which specialises in recognition, extraction and processing of image data to simulate human vision. '
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Artificial Intelligence'
+Tags:
+  - 'AI'
+  - 'Image Classification Model'
+  - 'Image Generation Model'
+  - 'Prompt Engineering'
+  - 'Image Labelling'
+  - 'Deep Learning'
+  - 'Classification'
+  - 'Facial Recognition'
+  - 'Machine Learning'
+CatalogContent:
+  - 'machine-learning'
+  - 'paths/data-science'
+---
+
+**Computer Vision** is the field of artificial intelligence that allows computers to analyse images, videos and 3D scans. The aim is to replicate human vision - *recognition of 3D and 2D objects*. Vision allows analysis and decision-making based on image data.
+
+*E.g... robot moving around the room and picking up objects based on a live video.*
+
+In addition, computer vision also is  the automation of tasks that require visual data. AI engineers aim to improve the performance of algorithms to make them better than humans at seeing objects.  
+
+*For autonomous driving, this could be stopping a car earlier than a human driver using distance detection from an obstacle.*
+
+## **Convolution** - How computers perceive images
+
+The convolution process is the decomposition of an image into different layers and features. This is achieved by specialised artificial neurons in **Convolutional Neuron Networks (CNNs)** layers, which use filters **kernels**  of a specific size that move across an image in a defined number of steps.  
+
+Kernel moves across an image:
+ ![CNN kernel moves across an image.](https://miro.medium.com/v2/resize:fit:990/1*hOI0jW3CcS_yuxcmJIYjKw.gif)
```

Comment:
We don't allow third party resurces. If you want to add a gif make sure you create it yourself then add it to the media folder.